superset-frontend/src/views/CRUD/data/dataset/AddDataset/LeftPanel/index.tsx

```diff
@@ -16,18 +16,227 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React from 'react';
-import { t } from '@superset-ui/core';
+import React, {
+  useEffect,
+  useState,
+  useMemo,
+  SetStateAction,
+  Dispatch,
+} from 'react';
+import { SupersetClient, t, styled, FAST_DEBOUNCE } from '@superset-ui/core';
+import { Input } from 'src/components/Input';
+import { Form } from 'src/components/Form';
+import { TableOption, Table } from 'src/components/TableSelector';
+import RefreshLabel from 'src/components/RefreshLabel';
+import Loading from 'src/components/Loading';
+import DatabaseSelector from 'src/components/DatabaseSelector';
+import { debounce } from 'lodash';
 import { EmptyStateMedium } from 'src/components/EmptyState';
+import { useToasts } from 'src/components/MessageToasts/withToasts';
+import { DatasetActionType, DatasetObject } from '../types';
+
+interface LeftPanelProps {
+  setDataset: Dispatch<SetStateAction<object>>;
+  schema?: string | undefined | null;
+  dbId?: number;
+}
+
+const LeftPanelStyle = styled.div`
+  ${({ theme }) => `
+  max-width: ${theme.gridUnit * 87.5}px;
+  padding: ${theme.gridUnit * 4}px;
+  height: 100%;
+  background-color: ${theme.colors.grayscale.light5};
+  position: relative;
+  .emptystate {
+    height: auto;
+    margin-top: 70px;
+  }
+  .refresh {
+    position: absolute;
+    top: ${theme.gridUnit * 43.25}px;
+    left: ${theme.gridUnit * 16.75}px;
+    span[role="button"]{
+      font-size: ${theme.gridUnit * 4.25}px;
+    }
+  }
+  .section-title {
+    margin-top: ${theme.gridUnit * 11}px;
+    margin-bottom: ${theme.gridUnit * 11}px;
+    font-weight: ${theme.typography.weights.bold};
+  }
+  .table-title {
+    margin-top: ${theme.gridUnit * 11}px;
+    margin-bottom: ${theme.gridUnit * 6}px;
+    font-weight: ${theme.typography.weights.bold};
+  }
+  .options-list {
+    overflow: auto;
+    position: absolute;
+    bottom: 0px;
+    top: 390px;
+    left: 13px;
```

Comment:
```suggestion
    top: ${theme.gridUnit * 97.5}px;
    left: ${theme.gridUnit * 3.25}px;
```
Unit values should stay within the Superset theme